public/index.html

```diff
@@ -9,7 +9,9 @@
       homescreen on Android. See https://developers.google.com/web/fundamentals/engage-and-retain/web-app-manifest/
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
+     <link href="https://fonts.googleapis.com/css?family=Raleway:500,700" rel="stylesheet">
```

Comment:
good job bringing in google fonts, I love seeing you keeping your old skills fresh and using them in projects like this.